examples/multisig/README.md

```diff
@@ -35,11 +35,14 @@ Sign again with addr2 (as threshold is set to 2)
 goal clerk multisig signprogram -L <path>/signed.msig -a addr2 -A <multisig_hash> -d ~/<data_directory>
 ```
 
-The logic sig file generated above is a raw binary file which is not readable. To disassemble the signed multisig binary 
+The logic sig file generated above is a `raw binary` file. You can save this file directly in `/assets` or you can disassemble this file to a readable format (command provided below) and then save it to `/assets`.
+
+ To disassemble the signed multisig binary 
 ```bash
 goal clerk compile -D <path>/signed.msig
 ```
-Paste the above output in `examples/multisig/assets/<file_name>.mlisg`
+Paste the above output in `examples/multisig/assets/<file_name>.msig`  
+Or directly paste the raw binary file in `/assets`
```

Comment:
the last line is not needed. Let's don't repeat ourselves.

Reply:
Ok